### PR TITLE
Like dislike

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**LIKE DISLIKE VIDEO**

_Style_

- Like page video card

_Functionility_

-  Likes context
- `get` watch-later data from API
- `post` video to watch-later API
- `delete` video from watch-later API

### CODE REVIEW

File: src/components/like-dislike/LikeListing.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.like-wrapper {
+  /* display: flex; */
```
Comment (deepak29-git) [inline src/components/like-dislike/LikeListing.css:2]: remove unnecessary code

**Review by sharathnair9999 — APPROVED** [review]

File: src/utilities/likes-utils.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      );
+      setLike(data.likes);
+    } catch (err) {
+      console.log(err);
+    }
+  }
+};
+
+const dislikeVideo = async (id, setLike) => {
+  const token = localStorage.getItem("token");
```
Comment (deepak29-git) [inline src/utilities/likes-utils.jsx:28]: you can make a function and use it everywhere

**Review by deepak29-git — APPROVED** [review]

File: src/components/like-dislike/LikeListing.css (hunk 1 above)
Comment (tanishk15sharma) [reply src/components/like-dislike/LikeListing.css:2]: yes

File: src/utilities/likes-utils.jsx (hunk 2 above)
Comment (tanishk15sharma) [reply src/utilities/likes-utils.jsx:28]: Yaa will make a function for this .

**Pull request merged**